packages/fluentui/react-northstar/src/components/Chat/chatItemContext.ts

```diff
@@ -1,9 +1,11 @@
 import { createContext } from '@fluentui/react-bindings';
+import type { ChatMessageLayout } from './ChatMessage';
```

Comment:
Could define a `ChatItemLayout` here instead and reverse the ownership, but it doesn't seem worth the complication/duplication. I'm indifferent 🤷 .

Reply:
It might be better to move it into this file to remove the circular dependency.

Reply:
Hit resolve too early, but it's been moved.